config/secrets.yml

```diff
@@ -11,10 +11,10 @@
 # if you're sharing your code publicly.
 
 development:
-  secret_key_base: 553149b03d6ba63414be540733245f323d241c1c9a37f3266d1216d93effd901fefb43f0782e4d2b739d61a3de1488c681a90435b22318114f638d9fc6e0f8c1
+  secret_key_base: 79252673745894b33179f707df65b3248f272f2e46bfa2078120f8c252f7e59331033872fe13be2b2259f0b01fc7d73d6d8e58c3b54969ff67d348bef69f959f
 
 test:
-  secret_key_base: 0c9e2ff02abb06b715fccfbd8a7fb390a12595cf1c4d8b58cae707903a17a4a58ea0762f4a2125da40cf59b211620b81e702e13d58c4a50e4591f48dcbf1cbaa
+  secret_key_base: 51273e88b99c66c8b3c1bfec07acd6decb4c824853659d0323bf3fd0927909240544f0f6a6bb7f2bed8e8761fd8439a43ae31f2a06571c8ad95282f73ed15c84
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
```

Comment:
Ahhhhhh these are secrets! They should not be on github! I'd recommend taking these down ASAP and doing some googling to make sure nothing bad happens. People build bots that scrape github looking  for secrets to take advantage of.
You could do env[SECRET_KEY_BASE] ...or whatever the syntax is lol. And stick your secrets in your .env file (which is gitignored)